code/src/components/app-box/AppBox.tsx

```diff
@@ -1,10 +1,14 @@
-import { ImgHTMLAttributes } from "react";
+import { ForwardedRef, HTMLProps, forwardRef } from "react";
 import Box, { BoxProps } from "@mui/material/Box";
 
-type AppBoxProps = BoxProps & ImgHTMLAttributes<HTMLImageElement>;
+type AppBoxProps<TElement> = BoxProps & HTMLProps<TElement>;
 
-const AppBox = (props: AppBoxProps) => {
-  return <Box {...props} />;
-};
+const AppBox = forwardRef(
+  <TElement,>(props: AppBoxProps<TElement>, ref: ForwardedRef<unknown>) => {
```

Comment:
I suggest to use `HTMLElement` instead `unknown` here

Reply:
That seems better, thanks, I fixed it as you mentioned